package.json

```diff
@@ -1,27 +1,48 @@
 {
   "name": "@nebulr-group/nblocks-ts-client",
-  "version": "5.4.1",
+  "version": "6.0.0-beta.1",
+  "exports": {
+    "./engine": {
```

Comment:
Not sure why this was added all of a sudden. Is this because you changed `tsconfig.json`?